tests/unit/test-cmd/test.run.js

```diff
@@ -1,6 +1,7 @@
 import path from 'path';
+import nodeFs from 'fs';
+import fs from 'fs/promises';
```

Comment:
Sometimes both APIs are used. For consistency, I let `fs` stay the promised API and used `nodeFs` for the other APIs. This name was actually used elsewhere too.